blocks/library/heading/test/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { query, getBlocks, unregisterBlock, setUnknownTypeHandler } from 'api';
+import { blockSettings } from '../';
+
+describe( 'heading block', () => {
+	afterEach( () => {
```

Comment:
>Maybe we should include the slug in the exported object? What do you think?

There's not really a strong reason not to. This is the approach @iseulde took with her block API in the single instance prototype ([example](https://github.com/WordPress/gutenberg/blob/349c8f3fee9370a1b8ed6285d7297c01c3eb6085/docs/tinymce-single/blocks/core/image/register.js#L72-L92)). My preference could be explained by it feeling natural to have a signature of `( key, options )` but this generally applies better in the case that `options` is optional (for us, it's not).

Conversely, most "register" functions in WordPress _do_ follow this signature pattern:

- https://developer.wordpress.org/reference/functions/register_post_type/
- https://developer.wordpress.org/reference/functions/register_rest_route/
- https://developer.wordpress.org/reference/functions/register_taxonomy/